OpenRA.Mods.Common/UtilityCommands/ExtractZeroBraneStudioLuaAPI.cs

```diff
@@ -0,0 +1,126 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Linq;
+using System.Reflection;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	// See https://studio.zerobrane.com/doc-api-auto-complete for reference
+	class ExtractZeroBraneStudioLuaAPI : IUtilityCommand
+	{
+		public string Name { get { return "--zbstudio-lua-api"; } }
+
+		public bool ValidateArguments(string[] args)
+		{
+			return true;
+		}
+
+		[Desc("Generate a ZeroBrane Studio Lua API and auto-complete description.")]
```

Comment:
I both agree and disagree with @Phrohdoh:
* We don't mention that `--docs` or `--lua-docs` should be redirected to md files, so I don't see why this should include a special mention here.
* The zerobrane section on the map scripting page is a good start for scripters, but we need documentation (or even better, automation) for updating https://github.com/pkulchenko/ZeroBranePackage/blob/master/openra.lua  somewhere that *we* are going to read.  I expect that integration will be forgotten and/or neglected by everyone except @Mailaender otherwise.